flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/plan/stream/sql/join/IntervalJoinTest.scala

```diff
@@ -64,7 +64,7 @@ class IntervalJoinTest extends TableTestBase {
        """.stripMargin)
 
   /** There should exist exactly two time conditions **/
-  @Test(expected = classOf[TableException])
+  @Test
```

Comment:
Those cases in IntervalJoinTest would fallback to the regular join because they don't satisfy the condition to convert to interval join. In the previous version, they would throw exception because input contains time attributes.
But in the new version, they could contain time attributes fields because they would materialized in RelTimeIndicatorConveter now.